andres-uzeda/homework7/mvc_task/src/App.java

```diff
@@ -0,0 +1,8 @@
+public class App {
+    public static void main(String[] args) {
+        Model m = new Model("Bob", "Sponge");
+        View v = new View("MVC Task");
```

Comment:
Here should send the list or any data structure you decided to use as parameter to controller constructor instead of User model